services/frontend/src/pages/HeroDetailPage.tsx

```
@@ -0,0 +1,243 @@
+function SynergyList({ rows, heroId }: { rows: SynergyResponse[]; heroId: number }) {
+  const navigate = useNavigate()
+  return (
+    <ol className="matchup-list">
+      {rows.map(r => {
+        const allyId = r.hero_id === heroId ? r.ally_id : r.hero_id
+        return (
+          <li
+            key={allyId}
+            className="matchup-row"
+            onClick={() => navigate(`/heroes/${allyId}/1`)}
+          >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)